src/side-navigation/side-navigation.scss

```diff
@@ -170,3 +171,41 @@
     }
   }
 }
+
+@mixin iui-side-navigation-submenu {
```

Comment:
How it should be minimized/maximized? In UX site an expander is above everything (sidenav + submenu): 
![image](https://user-images.githubusercontent.com/36186912/132341162-d48b0090-43b5-43d8-a1c3-fcf5aa39e118.png)

Reply:
That's a good point, I kind of overlooked that. 🤔 I'm kind of liking the min/max button only expanding the width of the side navigation because it's not really expanding/collapsing anything within the submenu.  I'm going to pass this idea around the UX team and see if there are any thoughts.

Reply:
I spoke with a couple of UXers about the placement of the expander button.  They are ok with restricting it to only the side navigation.  In fact, some of their designs in Axure already have it this way:
![Screen Shot 2021-09-15 at 11 52 24 AM](https://user-images.githubusercontent.com/849817/133466899-e466b05d-6069-4ed7-a806-bdafed2b144a.png)